CharlieBackend.Api.UnitTes/LessonServiceTests.cs

```diff
@@ -11,104 +11,120 @@
-            mentorReposutoryMock.Setup(x => x.GetMentorByIdAsync(createLessonDTO.MentorId)).ReturnsAsync(new Mentor { Id = createLessonDTO.MentorId });
+        private static CreateLessonDto AddCreateLessonDto()
+        {
+            return new CreateLessonDto
+            {
+                ThemeName = themeName,
+                MentorId = mentorId,
+                StudentGroupId = studentGroupId,
+                LessonDate = lessonDate,
+                LessonVisits = CreateVisitDto()
+            };
+        }
 
-            var studentGroupRepositoryMock = new Mock<IStudentGroupRepository>();
-            studentGroupRepositoryMock.Setup(x => x.GetByIdAsync(studentGroup.Id)).ReturnsAsync(studentGroup);
-            studentGroupRepositoryMock.Setup(x => x.GetGroupStudentsIds(studentGroup.Id)).ReturnsAsync(new List<long?> { 11, 14 });
+        private static StudentGroup CreateStudentGroup()
+        {
+            List<StudentOfStudentGroup> studentOfStudentGroup = new List<StudentOfStudentGroup>
+            {
+                new StudentOfStudentGroup { StudentGroupId = studentGroupId, StudentId = visitStudentIdPresenceTrue },
+                new StudentOfStudentGroup { StudentGroupId = studentGroupId, StudentId = visitStudentIdPresenceFalse }
+            };
+            return new StudentGroup() { Id = 3, StudentsOfStudentGroups = studentOfStudentGroup };
+        }
+        private void MockEntities()
+        {
+            _studentGroupRepositoryMock.Setup(x => x.GetByIdAsync(CreateStudentGroup().Id)).ReturnsAsync(CreateStudentGroup());
 
-            var visitRepositoryMock = new Mock<IVisitRepository>();
+            _studentGroupRepositoryMock.Setup(x => x.GetGroupStudentsIds(CreateStudentGroup().Id))
+                .ReturnsAsync(new List<long?> { visitStudentIdPresenceTrue, visitStudentIdPresenceFalse });
 
+            _unitOfWorkMock.Setup(x => x.LessonRepository).Returns(_lessonRepositoryMock.Object);
+            _unitOfWorkMock.Setup(x => x.ThemeRepository).Returns(_themeRepositoryMock.Object);
+            _unitOfWorkMock.Setup(x => x.MentorRepository).Returns(_mentorRepositoryMock.Object);
+            _unitOfWorkMock.Setup(x => x.StudentGroupRepository).Returns(_studentGroupRepositoryMock.Object);
+            _unitOfWorkMock.Setup(x => x.VisitRepository).Returns(_visitRepositoryMock.Object);
 
-            _unitOfWorkMock.Setup(x => x.LessonRepository).Returns(lessonRepositoryMock.Object);
-            _unitOfWorkMock.Setup(x => x.ThemeRepository).Returns(themeRepositoryMock.Object);
-            _unitOfWorkMock.Setup(x => x.MentorRepository).Returns(mentorReposutoryMock.Object);
-            _unitOfWorkMock.Setup(x => x.StudentGroupRepository).Returns(studentGroupRepositoryMock.Object);
-            _unitOfWorkMock.Setup(x => x.VisitRepository).Returns(visitRepositoryMock.Object);
 
             _currentUserServiceMock = GetCurrentUserAsExistingStudent();
+        }
+        [Fact]
```

Comment:
```suggestion
        }
        
        [Fact]
```
Adding 1 empty line between test methods

Reply:
done